scripts/includes/common.sh

```diff
@@ -1,3 +1,9 @@
+function get_home() {
+  USER=$1
```

Comment:
let's use local variable here, instead of overriding `$USER`

Reply:
Good point :+1: